cmd/tester/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM golang:1.9.2 as builder
+RUN mkdir -p /go/src/github.com/openfaas-incubator/connector
+WORKDIR /go/src/github.com/openfaas-incubator/connector
+
+COPY main.go    .
+
+RUN go get -d -v ./...
```

Comment:
Please remove `go get`

It should use vendoring via `dep` or the correct path.